crates/bevy_reflect/src/type_registry.rs

```diff
@@ -406,6 +406,15 @@ impl TypeRegistry {
     pub fn iter_mut(&mut self) -> impl Iterator<Item = &mut TypeRegistration> {
         self.registrations.values_mut()
     }
+
+    /// Returns an iterator over the [`TypeRegistration`]s with their corresponding [`TypeData`]
```

Comment:
Can you add a small doc test here? It's not immediately clear to the reader what `T` might be.